packages/smooth_app/pubspec.lock

```diff
@@ -1157,5 +1157,5 @@ packages:
     source: hosted
     version: "3.1.0"
 sdks:
-  dart: ">=2.15.1 <3.0.0"
+  dart: ">=2.16.0-100.0.dev <3.0.0"
```

Comment:
Very surprising, as we have this in the `pubspec.yaml`:
```yaml
environment:
  sdk: ">=2.15.1 <3.0.0"
```
I recommend a `flutter clean` then `flutter run`.
That said, I've never really understood the purpose of `pubspec.lock`.